massa-models/src/denunciation.rs

```diff
@@ -257,6 +258,18 @@ impl Denunciation {
         }
     }
 
+    /// Check if denunciation has expired given a slot period (assuming the slot is final)
```

Comment:
no need to assume the slot is final, this is an unrelated concept. Is_expired is also used when checking if the denunciation can still be executed when checking a non-final block

Reply:
I've clarified the documentation and mentions the usage so the reader knows when it is used.